Update manage-proxy.adoc

amagnus opened the pull request.

Commit:
Update manage-proxy.adoc

sundippatel committed:
Update manage-proxy.adoc

Fix typo

sundippatel committed:
Update manage-proxy.adoc

- Fix broken link
- Added procedure to update rtf cluster for http proxy

Commit:
Update version of rtfctl artifact

Commit:
Update manage-proxy.adoc

amagnus commented on modules/ROOT/pages/manage-proxy.adoc:
```diff
@@ -7,40 +7,53 @@ This topic describes how to set or update proxy settings used by Runtime Fabric
+
+=== Run script on each node
+
+. Copy the `set_dockerd_proxy.sh` from the controller node used to install Runtime Fabric located under `/opt/anypoint/runtimefabric/scripts/` to each controller and worker node under the same location (`/opt/anypoint/runtimefabric/scripts/`).
```
should we assume that they have the latest installer here?

sundippatel replied:
Good point. We'll need to provide this script in a location (similar to the Mule license key update script) for customers running on older versions to obtain.

sundippatel replied:
Added a note to guide customers to open a support ticket if unable to locate the script.

amagnus replied:
👍

sundippatel committed:
Update manage-proxy.adoc

Added note to ensure password is URL encoded.

sundippatel committed:
Update manage-proxy.adoc

Added note to request a support ticket in case customer is unable to locate the script to update proxy settings on each node.

sundippatel committed:
Update manage-proxy.adoc

Updated additional link to rtfctl

valkyrie69 committed:
Minor edits to conform to MuleSoft style
